scripts/sem_bleaching_map.py

```diff
@@ -1,33 +1,42 @@
 #!/usr/bin/env python
 # -*- coding: utf-8 -*-
-'''
+"""
 Created on 20 Mar 2014
 
-@author: Éric Piel
+@author: Éric Piel, modified by Aaro Vakevainen, modified by Lennard Voortman
```

Comment:
Who are Aaro Vakevainen and Lennard Voortman? Maybe we can write there affiliation behind there name in brackets.
@VictoriaDel Shouldn't your name also be here/

Reply:
When working on this script, Lennard used to work in Delmic, while Aaro was a TUDelft student (I think).